editor/editor_properties_array_dict.cpp

```diff
@@ -1116,7 +1197,7 @@ void EditorPropertyDictionary::update_property() {
 			if (i == amount) {
 				PanelContainer *pc = memnew(PanelContainer);
 				property_vbox->add_child(pc);
-				pc->add_theme_style_override(SNAME("panel"), get_theme_stylebox(SNAME("DictionaryAddItem"), SNAME("EditorStyles")));
+				pc->add_theme_style_override(SNAME("panel"), get_theme_stylebox(SNAME("DictionaryAddItem")));
```

Comment:
Type should remain here otherwise it doesn't work.

Reply:
On the contrary with it, would fail. The subtility here is that before `DictionaryAddItem` was a stylebox from `EditorStyles` but now there is one for each depth just as inspector background so when `_update_property_bg` is called I store (through override) which stylebox is appropriate in `DictionaryAddItem` and that is what I'm fetching here.

Reply:
Then that's an unnecessarily complicated way to do it. Create a theme cache struct and store those items in members of that struct (look how to implement one in any control node).

Reply:
Hmm, I guess that's how it's been implemented before. Well, can wait for a refactoring pass later then.